evutil.c

```diff
@@ -1786,9 +1786,18 @@ evutil_dup_addrinfo_(struct evutil_addrinfo *ai)
 	for (; ai; ai = ai->ai_next) {
 		int len = sizeof(struct evutil_addrinfo) + ai->ai_addrlen;
 		struct evutil_addrinfo *n = mm_calloc(1, len);
+		if (!n) {
```

Comment:
This may leave some leaks in case of failure will happen not on the first iteration of a loop

Reply:
This is inside a for loop, and it's possible that memory allocated in previous iterations has not been released.

Reply:
> This may leave some leaks in case of failure will happen not on the first iteration of a loop

I will remove the modifications to this function later.